game/game/playercontrol.cpp

```diff
@@ -430,6 +430,7 @@ void PlayerControl::marvinF8(uint64_t dt) {
   pl.clearSpeed();
   pl.quitIneraction();
   pl.setAnim(AnimationSolver::Idle);
+  pl.applyAirPhysics();
```

Comment:
Looking into this change: I'm not sure about this design choose. So far all physics state bits were encapsulated in `MoveAlgo`, and been read-only for rest of code. Why `invalidatePhysics`-kind approach was discarded?